src/main/java/htsjdk/samtools/util/IOUtil.java

```diff
@@ -245,7 +245,7 @@ public static void deletePaths(final Iterable<Path> paths) {
         for (final Path p : paths) {
             boolean deleted;
             try {
-                deleted = Files.delete(p);
+                Files.delete(p);
```

Comment:
Now you can get rid of `deleted` and move the `println()` into the `catch` clause, and output the exception as well.

Reply:
True, I forgot. My bad!